### PR TITLE
:zap: opt unuse label

### PR DESCRIPTION
**Pull request by songxiaosheng**
## What is the purpose of the change

remove unuse label : host、ip、gitcommitid
we can get it（host)  when discover host such as prometheus use $instance

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Review by AlbumenJ — CHANGES_REQUESTED** [review]
Can we only add `commitId` and `hostname` tag only in `dubbo_application_info_total`

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java [hunk 1]
```diff
@@ -59,25 +59,30 @@
...
         tags.put(TAG_APPLICATION_NAME, applicationModel.getApplicationName());
         tags.put(TAG_APPLICATION_MODULE, applicationModel.getInternalId());
         tags.put(TAG_APPLICATION_VERSION_KEY, version);
-        tags.put(MetricsKey.METADATA_GIT_COMMITID_METRIC.getName(), commitId);
         if (CollectionUtils.isNotEmptyMap(extraInfo)) {
             tags.putAll(extraInfo);
         }
         return tags;
     }
+    public static Map<String, String> applicationPrivateTags(ApplicationModel applicationModel, Map<String, String> tags) {
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:80]:
```suggestion
    public static Map<String, String> hostInfoTags(ApplicationModel applicationModel, Map<String, String> tags) {
```

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java [hunk 2]
```diff
@@ -59,25 +59,33 @@
...
         return applicationTags(applicationModel, null);
     }
 
     public static Map<String, String> applicationTags(ApplicationModel applicationModel, @Nullable Map<String, String> extraInfo) {
         Map<String, String> tags = new HashMap<>();
-        tags.put(TAG_IP, getLocalHost());
-        tags.put(TAG_HOSTNAME, getLocalHostName());
         tags.put(TAG_APPLICATION_NAME, applicationModel.getApplicationName());
         tags.put(TAG_APPLICATION_MODULE, applicationModel.getInternalId());
         tags.put(TAG_APPLICATION_VERSION_KEY, version);
```
Comment (AlbumenJ) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:74]: `Version` key can be moved into `gitTags`

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Review by AlbumenJ — APPROVED** [review]

File: dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java [hunk 3]
```diff
@@ -59,25 +59,33 @@
...
         }
         return tags;
     }
+    public static Map<String, String> gitTags(Map<String, String> tags) {
+        tags.put(MetricsKey.METADATA_GIT_COMMITID_METRIC.getName(), commitId);
+        tags.put(TAG_APPLICATION_VERSION_KEY, version);
+        return tags;
+    }
+    public static Map<String, String> hostTags( Map<String, String> tags) {
+        tags.put(TAG_IP, getLocalHost());
```
Comment (bert82503) [inline dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:85]: 服务方法的指标数据，ip标签为何移走了？是怎么考量的？
某个方法耗时长，怎么反推定位到是某个ip实例触发的？通过traceId定位么

Comment (songxiaosheng) [reply dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:85]: ip可以直接查指标来源的

Comment (bert82503) [reply dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsSupport.java:85]: 明白了，感谢！👍🏻

**Pull request merged**